components/renku_data_services/migrations/versions/9df92d455b11_make_work_and_mount_dir_nullable.py

```diff
@@ -1,7 +1,7 @@
 """make work and mount dir nullable
 
 Revision ID: 9df92d455b11
-Revises: 1ef98b967767
+Revises: ea52d750e389
```

Comment:
?

Reply:
This wasn't matching the version in the code: https://github.com/SwissDataScienceCenter/renku-data-services/blob/c57ec3ad9869f58ddbdeab99046ec9efba567c4e/components/renku_data_services/migrations/versions/9df92d455b11_make_work_and_mount_dir_nullable.py#L14